paulshryock/paul-shryock
# #29: Purge unused CSS, fix JavaScript linting

## Summary

There's no need to bloat production CSS assets. Purge unused CSS.

## Testing
- Run `npm start`
- Verify that unused CSS is purged from production CSS

## Issue(s)

Closes #29

## Changelog

### Added
Purge unused CSS. [#29]

### Fixed
- Fix JavaScript linting.

## Release Checklist
